engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/engine/embedded/EmbeddedDeploymentManager.scala

```diff
@@ -99,11 +99,11 @@ class EmbeddedDeploymentManager(
             throwInterpreterRunExceptionsImmediately = true
           )
         }
-      case command: CancelDeploymentCommand => cancelDeployment(command)
-      case command: CancelScenarioCommand   => cancelScenario(command)
-      case command: TestScenarioCommand     => processTestActionCommand(command)
-      case _: StopDeploymentCommand | _: StopScenarioCommand | _: MakeScenarioSavepointCommand |
-          _: CustomActionCommand =>
+      case command: DMCancelDeploymentCommand => cancelDeployment(command)
+      case command: DMCancelScenarioCommand   => cancelScenario(command)
+      case command: DMTestScenarioCommand     => processTestActionCommand(command)
+      case _: DMStopDeploymentCommand | _: DMStopScenarioCommand | _: DMMakeScenarioSavepointCommand |
```

Comment:
To be consistent: processTestScenario?

Reply:
I guess that you want to left the comment next to the line above?  Take a look that methods handling concrete command don't have neighter process* nor *command. I think that for sake of consistency it should be rather just `testScenario`. WDYT? I also changed `validate` -> `validateScenario` to be more explicit.